src/parameterized_tendencies/microphysics/precipitation.jl

```diff
@@ -213,15 +234,14 @@ function compute_precipitation_cache!(Y, p, ::Microphysics1Moment, _)
     compute_precipitation_sources!(
         ᶜSᵖ,
         ᶜSᵖ_snow,
-        ᶜSqₜᵖ,
+        ᶜSqₗᵖ,
+        ᶜSqᵢᵖ,
         ᶜSqᵣᵖ,
         ᶜSqₛᵖ,
-        ᶜSeₜᵖ,
         Y.c.ρ,
-        ᶜqᵣ,
-        ᶜqₛ,
+        q_rai,
+        q_sno,
```

Comment:
Just note that these could be negative.

Reply:
That should be fine. I'll double check just in case but all microphysics functions return zero if one of the tracers is negative. And all the limiters inside ClimaAtmos precipitation are written anticipating tracers can be negative too. No trust!